XamlControlsGallery/AllControlsPage.xaml.cs

```diff
@@ -31,7 +31,8 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
         {
             var menuItem = NavigationRootPage.Current.NavigationView.MenuItems.Cast<Microsoft.UI.Xaml.Controls.NavigationViewItem>().ElementAt(1);
             menuItem.IsSelected = true;
-            NavigationRootPage.Current.NavigationView.Header = menuItem.Content;
+            NavigationRootPage.Current.NavigationView.Header = string.Empty;
```

Comment:
Just out of curiosity: Do we need this line? And if so what exactly does it do?  
I have commented it out and I saw no difference in the AllControlsPage. 😅

Reply:
I think whether or not this line does anything depends on whether or not the Header text is already set. The NavigationView Header only shows up on narrow widths, but I think that without this line we run the risk of showing the wrong header depending on the path the user took to get to this page.

Reply:
Oh okay, I see. We should leave it that way than just to be save.